### ISSUE TITLE
`force-expand-multiline` not working correctly with handlebars block statements

### ISSUE THREAD

**Issue by brokenalarms** [beautify-web/js-beautify#1537]
# Description
> This is the default template for bug reports
>
> NOTE: 
> * Do not include screenshots! This library is a text processor, we need text inputs and outputs for debugging and fixing issues. 
> * Check the list of open issues before filing a new issue. 

# Input
The code looked like this before beautification:
```
{{#creative-edit-panel
  isEdit=isEdit
  isValid=variables.validations.isValid
}}
```

# Expected Output
The code should have looked like this after beautification:
```
{{#creative-edit-panel
  isEdit=isEdit
  isValid=variables.validations.isValid
}}
```

# Actual Output
The  code actually looked like this after beautification:
```
{{#creative-edit-panel isEdit=isEdit
  isValid=variables.validations.isValid
}}
```

# Steps to Reproduce

## Environment
OS:

## Settings
Example:
```json
{
    "indent_size": 4,
    "indent_char": " ",
    "indent_level": 0,
    "indent_with_tabs": false,
    "preserve_newlines": true,
    "max_preserve_newlines": 10,
    "jslint_happy": false,
    "space_after_anon_function": false,
    "brace_style": "collapse,preserve-inline",
    "keep_array_indentation": false,
    "keep_function_indentation": false,
    "space_before_conditional": true,
    "break_chained_methods": false,
    "eval_code": false,
    "unescape_strings": false,
    "wrap_line_length": 0
}
```

**Comment by bitwiseman** [comment]
This is somewhere between a bug and enhancement.  This handlebars behavior has never been great and it is possible to argue that it is better than it has ever been, but part of how that was accomplished meant not parsing handlebars tag contents as attributes, but just as text.  See #1469.  

If this was working before, it was likely coincidence and caused other issues such as #1040.  

It looks like someone did a drive reporting of this a while back (#795). Thanks for the solid  bug report. 

Side note: `"html.format.contentUnformatted": "pre,code,textarea,{{#",` is an interesting attempt but it won't do what you're shooting for, sorry.  For the example above, you'd need to use `creative-edit-panel` instead of `{{#` and it would need to be added to the `unformattted` list, which would have other side effects. 😞This needs to be fixed internally not by additional setting.

**Comment by bitwiseman** [comment]
What editor/plugin are you using? 
In the latest 1.8.x beautifier, you shouldn't need to have `wbr` in the `unformatted` list.

**Comment by bitwiseman** [comment]
@brokenalarms  
I'm trying to understand where the attributes rule applies.  Is it only for `name=value` or can it be other cases such as just `name`.    

So, for `{{#each items}}` the "items" should not be treated as an attribute right?  Or should it? 

Same question for `{{#each child as |childValue childKey|}}`.

**Comment by bitwiseman** [comment]
@brokenalarms  
Okay, I looked at https://handlebarsjs.com/ and I get something now.  
The items you'd like to be treated like attributes are in fact _parameters_ to helpers  .  I understand they become attributes in the output, but they really aren't attributes in handlebars. Which is why figuring out how they should behave is problematic. 

I would love help implementing better behavior but it may need to be an opt-in setting to ensure it doesn't break other users.

**Comment by brokenalarms** [comment]
Yes, ideally it should only be the `prop=value` lines that are formatted as these since they are more generally in the context of "block level helpers" which are frequently components and therefore more analogous to HTML web components which would be similarly formatted.

```
{{#block}}{{/block}}
```

Having it for built in helpers like `{{#each child as |childValue childKey|}}` would not give the desired effect since these are purely parameters to helpers and not thought of in the same way as passing props/attributes to components - it was actually a previous issue I lodged that the formatting for these got messed up and became
```each
child
as
childKey
```
Which was another earlier blocker to having autosave for hbs files switched on in VSCode.

It's now at the point where I can actually have autosave on and it's close enough except for not wrapping the first attribute to the hbs components. It does wrap the second onwards though, and doesn't wrap `each` as desired, so it does seem like it's treating them differently and understands wrapping for component props but is just not applying to the first one.

**Comment by brokenalarms** [comment]
Sounds good, thanks for looking into this!
